dataprofiler/profilers/column_profile_compilers.py

```diff
@@ -194,6 +209,62 @@ def profile(self):
             )
         return profile
 
+    def diff(self, other, options=None):
+        """
+        Finds the difference between 2 compilers and returns the report
+        
+        :param other: profile compiler finding the difference with this one.
+        :type other: ColumnPrimitiveTypeProfileCompiler
+        :return: difference of the profiles
+        :rtype: dict
+        """
+        # Call super for compiler instance check
+        super().diff(other)
```

Comment:
I find it strange to call it but not utilize it's return even if it is an empty dict

Reply:
utilized now